### DISCUSSION TITLE
Student developer pack

### DISCUSSION THREAD

**Discussion by Aaditya-Siddharth** [community/community#41678] [GitHub Education]
Why am I not able to land in Student Developer pack page and make a application?

**Comment by jung-youjin** [comment]
Have you tried validating your status as a student with your school email address domain? If you successfully do so, you will get access to the Student Developer pack! 📚 🎒

**Comment by samus-aran** [comment]
Hi @Aaditya-Siddharth  👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
